<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWidget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>981</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Iniciar">
      <property name="text">
       <string>CONECTAR</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Finalizar">
      <property name="text">
       <string>DESCONECTAR</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>133</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Serial port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comPort"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>686</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>D0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>D1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>D2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>D3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>D4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>D5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>D6</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>D7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="8">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="D0">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D1">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D2">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D3">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D4">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D5">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D6">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D7">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>681</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLCDNumber" name="ADC_2"/>
    </item>
    <item row="1" column="3">
     <widget class="QLCDNumber" name="ADC_1"/>
    </item>
    <item row="1" column="7">
     <widget class="QLCDNumber" name="ADC_7"/>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>ADC5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLCDNumber" name="ADC_5"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ADC0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLCDNumber" name="ADC_0"/>
    </item>
    <item row="1" column="4">
     <widget class="QLCDNumber" name="ADC_3"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ADC2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>ADC3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>ADC7</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>ADC6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLCDNumber" name="ADC_4"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ADC1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLCDNumber" name="ADC_6"/>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>ADC4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>261</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>DAC0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>DAC1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDial" name="dialDAC0"/>
    </item>
    <item row="2" column="1">
     <widget class="QDial" name="dialDAC1"/>
    </item>
    <item row="3" column="0">
     <widget class="QLCDNumber" name="DAC_0"/>
    </item>
    <item row="3" column="1">
     <widget class="QLCDNumber" name="DAC_1"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Digital Input</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>681</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>D0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>D1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>D2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_34">
        <property name="text">
         <string>D3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>D4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_36">
        <property name="text">
         <string>D5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_37">
        <property name="text">
         <string>D6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_38">
        <property name="text">
         <string>D7</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLCDNumber" name="inputD0"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="inputD1"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="inputD2"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="inputD3"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="inputD4"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="inputD5"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="inputD6"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="inputD7"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Digital Output</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Iniciar</tabstop>
  <tabstop>Finalizar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dialDAC0</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_20</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>747</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialDAC1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_21</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>894</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>882</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
